src/app/view.component.html

```diff
@@ -1,25 +1,51 @@
+
+<nav mat-tab-nav-bar >
```

Comment:
This nav bar takes up too much real-estate up on top. If you guys are going to keep this thing in here, you need to shrink it.